worker-turso-ts/src/index.ts

```diff
@@ -0,0 +1,142 @@
+// import { createClient } from '@libsql/client/http';
+import { Client as LibsqlClient, createClient } from '@libsql/client/web';
+import { Router, RouterType } from 'itty-router';
+
+export interface Env {
+	LIBSQL_DB_URL?: string;
+	LIBSQL_DB_AUTH_TOKEN?: string;
+	router?: RouterType;
+}
+
+export default {
+	async fetch(request: Request, env: Env): Promise<Response> {
+		if (env.router === undefined) {
+			env.router = buildRouter(env);
+		}
+
+		return env.router.handle(request);
+	},
+};
+
+function buildLibsqlClient(env: Env): LibsqlClient {
+	const url = env.LIBSQL_DB_URL?.trim();
+	if (url === undefined) {
+		throw new Error('LIBSQL_DB_URL env var is not defined');
+	}
+
+	const authToken = env.LIBSQL_DB_AUTH_TOKEN?.trim();
+	if (authToken === undefined) {
+		throw new Error('LIBSQL_DB_AUTH_TOKEN env var is not defined');
+	}
+
+	return createClient({ url, authToken });
+}
+
+function buildRouter(env: Env): RouterType {
+	const router = Router();
+
+	router.get('/tasks', async () => {
```

Comment:
it must be passed a phone number! (the number of the mobile of the sales agent)

Reply:
You will need to add a UI to get the user's phone number. Just put it in a settings tab